feedhenry/fh-ios-swift-sdk
# Resolving issue with new Xcode build system

# What
This PR resolves an issue with the Xcode "New Build System" which was introduced in Xcode 10.

The error resolved during the build is:

```
error: Multiple commands produce '/Users/xxx/Library/Developer/Xcode/DerivedData/sync-ios-app-eumnttjximgtgqhcwvuholnkansu/Build/Products/Debug-iphonesimulator/FeedHenry/FeedHenry.framework/Headers/FeedHenry-Swift.h':

1) Target 'FeedHenry' (project 'Pods') has copy command from '/Users/xxx/projects/mta/PH-Test-Sync-Sync-iOS--Swift--App/Pods/FeedHenry/FeedHenry/FeedHenry-Swift.h' to '/Users/xxx/Library/Developer/Xcode/DerivedData/sync-ios-app-eumnttjximgtgqhcwvuholnkansu/Build/Products/Debug-iphonesimulator/FeedHenry/FeedHenry.framework/Headers/FeedHenry-Swift.h'

2) Target 'FeedHenry' (project 'Pods'): Ditto /Users/xxx/Library/Developer/Xcode/DerivedData/sync-ios-app-eumnttjximgtgqhcwvuholnkansu/Build/Products/Debug-iphonesimulator/FeedHenry/FeedHenry.framework/Headers/FeedHenry-Swift.h /Users/xxx

```

# Why
The "legacy build system" will most likely be removed as an option in Xcode 11, rendering apps build with the RHMAP SDK unbuildable

# How
The issue relates to the naming of the file FeedHenry-Swift.h.  Renaming this to FeedHenry-Sw.h and updating references resolves the issue.
Not sure why, but I think this is because the target is called FeedHenry and Xcode is auto generating a FeedHenry-Swift.h file


# Verification Steps
Open a project in Xcode 10.1 and select "New Build System", the project should build successfully. 


## Checklist:

- [ ] Changes have been successfully verified by Feedhenry team member
